cmd/engine/engine.go

```diff
@@ -214,7 +217,9 @@ func doCmd(_ *cobra.Command, _ []string) (err error) {
  $$ |      $$ |$$ |  $$ |$$ |  $$ |$$ |  $$ |$$ |  $$ |$$\ $$ |  $$ |$$ |
  \$$$$$$\  $$ |\$$$$$   |\$$$$$   |\$$$$$$  |$$ |  \$$$   |\$$$$$   |$$ |
   \_______|\__| \______/  \______/  \_______|\__|   \____/  \______/ \__|
- `)
+ 
+  Version %s\n
+  `, version)
```

Comment:
can we somehow set a default value in case this is not linked (and not a release) to something like (devel)? Or even better construct a "fake" version out of the git hash?